src/Umbraco.Web/Dashboards/ContentDashboard.cs

```diff
@@ -9,19 +12,30 @@ public class ContentDashboard : IDashboard
     {
         public string Alias => "contentIntro";
 
-        public string[] Sections => new [] { "content" };
+        public string[] Sections => new[] { "content" };
 
         public string View => "views/dashboard/default/startupdashboardintro.html";
 
         public IAccessRule[] AccessRules
         {
             get
             {
-                var rules = new IAccessRule[]
+                IAccessRule[] rules;
```

Comment:
Any reason not just to allow all user groups? If the end user has added custom user groups, these are not allowed to see the dashboard now. I think a simple appsetting in webconfig is enough, just to tell all user groups are allowed to see the dashboard

Reply:
Good idea, will refactor!